# replacing /etc/mobile-config-firefox with /app/etc/mobile-config-firefox
# we heed it because flatpak uses /app/etc instead of /etc
--- a/./src/mobile-config-autoconfig.js
+++ b/./src/mobile-config-autoconfig.js
@@ -102,7 +102,7 @@
     // We're converting our path to a nsIFile so it can be consumed by
     // autoRegister below.
     const chromeManifest =
-        new FileUtils.File("/usr/lib/mobile-config-firefox/chrome.manifest");
+        new FileUtils.File("/app/usr/lib/mobile-config-firefox/chrome.manifest");

     if(chromeManifest.exists()){
         Components.manager.QueryInterface(Ci.nsIComponentRegistrar)
